acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -249,6 +249,17 @@ context('Acceptance tests', () => {
             cy.get('label[for=request_new_tax_number]').should('not.be.checked')
         })
 
+        it('Enter disability', () => {
+            cy.visit('/lotse/step/person_a_has_disability?link_overview=True')
+
+            // Set disability status to yes
+            cy.get('#person_a_has_disabilityyes').check().should('be.checked')
+            cy.get(submitBtnSelector).click()
+
+            // Select pauschbetrag
+            cy.get('#person_a_requests_pauschbetragyes').check().should('be.checked')
+        })
```

Comment:
I meant a whole runthrough og the whole app with disabilities set - not only parts of the steps - we are already testing this in the integration tests. But maybe this isn't even such a good idea overall as Niko always says the acceptance test should only test one happy path? I'm really not sure. But either, we add a complete runthrough or delete this as well imo.

Reply:
Deleted it :) If the acceptance tests only tests the happy path I'm fine with that.